lib/src/modelinfo/modelproperty.dart

```diff
@@ -98,6 +98,8 @@ class ModelProperty {
         hasFlag(OBXPropertyFlags.INDEX_HASH64);
   }
 
+  bool get isRelation => type == OBXPropertyType.Relation;
```

Comment:
The `prop.type == OBXPropertyType.Relation` check is still used in some other places. Should these be replaced as well?

Reply:
replaced